learning-bitmark/quick-start/simple-solution-for-node-setup.md

```diff
@@ -1 +1,67 @@
 #  [Learning Bitmark / Quick start] - Simple solution for node setup
+
+### Start your bitmark-node
+This section will quide you to start a bitmark docker node. The **docker** node, also called **bitmark-node**, is a bitmark full-node. A full-node can generate blocks, mine the blocks and do verification of payments. 
+
+### Assumptions
++ [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) is installed in your system
++ [docker](https://docs.docker.com/install/) is installed in your system
+    +  Windows user run docker on Hyper-V
++ you [log in](https://docs.docker.com/engine/reference/commandline/login/) your docker with [docker hub]((https://hub.docker.com/)) account
++ you have and know your **public IP** address
++ If you are behind a firewall, you know how to setup a **port-forwarding**
++ you have **a bitcoin and a litecoin address** for recieving mining reward
+
+## Installation
+
+### Step 0: Preparation
++ Have your **Public IP** ready
++ Setup port-forwarding if nessasary
```

Comment:
Readers might not remember what makes it necessary to set up port forwarding?

Reply:
1. I think the list under `Assumptions` is more about `Prerequites`. 
2. Why do we need `git` installed?

Reply:
@nodestory as discussion change to
```+ You may need to setup your network, ie. port forwarding, in order to allow other bitmark nodes to communicate with your bitmark node.```
@vienlt  
1. I can change to "Prerequisites" . Libra uses Assumptions. They may have a subtle difference but I think we don't need to argue and maybe leave it to Technical writer. 
2. Remove git

Reply:
@vienlt  You also may consider We have a preparation step so is a prerequisite a better word?  I'll let you decide